ros/src/tl_detector/light_classification/ssd_tl_classifier.py

```diff
@@ -34,37 +47,112 @@ def load_graph(graph_file):
                 serialized_graph = fid.read()
                 od_graph_def.ParseFromString(serialized_graph)
                 tf.import_graph_def(od_graph_def, name='')
-
         return graph
 
+    def filter_boxes(self, boxes, scores, classes):
```

Comment:
Since we use so-called "protected" methods in classifier classes, it is reasonable to make this method also protected: `_filter_boxes`.